docs/sources/setup-grafana/configure-grafana/proxy.md

```diff
@@ -0,0 +1,50 @@
+---
+aliases:
+  - ../features/datasources/proxy/
+description: Proxy datasource connections through a secure socks proxy.
+keywords:
+  - proxy
+  - guide
+title: Proxying data source connections
+weight: 1110
+---
+
+# Proxying data source connections
```

Comment:
@stephaniehingtgen , we typically avoid gerunds in topic titles. Can this be "Proxy a data source connections"? Or "Configure a data source connection proxy"?

Reply:
For sure! Changed it to `Configure a data source connection proxy`